src/types/transfer.ts

```
@@ -0,0 +1,307 @@
+import { ethers, BigNumber } from "ethers";
+import { getRpcProvider } from "../utils/get-fastest-provider";
+import { getPrivateKey } from "../utils/keys";
+import { Context, Logger } from "../types/context";
+import { PermitReward, PERMIT2_ADDRESS } from "../types";
```

ESLint

[error] 'PERMIT2_ADDRESS' is defined but never used. Allowed unused vars must match /^_/u.

(`@typescript-eslint/no-unused-vars`)

---

[error] Refactor this function to reduce its Cognitive Complexity from 21 to the 15 allowed.

(sonarjs/cognitive-complexity)